src/apple/code-tools.ts

```diff
@@ -97,6 +97,7 @@ export function addCodeSnippetToProject(
   if (!match) {
     const swiftUIMatch = swiftUIRegex.exec(fileContent);
     if (!swiftUIMatch) {
+      // This branch is not reached, because we already checked for SwiftUI in isAppDelegateFile
```

Comment:
`m`: Why do we need the check then?

Reply:
The compiler requires the null check, I can probably get rid of it, but only by refactoring the implementation. This PR focuses on adding initial test coverage